UPGRADE-14.0.md

```diff
@@ -1036,6 +1036,14 @@ Follow the instructions in relevant sections, e.g. `shopsys/coding-standards` or
         )
     ```
 
+    -   [features moved](#movement-of-features-from-project-base-to-packages) to the `framework` package:
+
+        -   `ProductFilterData` and part of related logic
```

Comment:
I would add one more thing:

- `ClassExtensionRegistry::getOtherClassesExtensionMap()` logic was moved to its parent class